src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -2950,7 +2950,18 @@ This package will be unloaded after the next Dynamo restart.</value>
     <value>Info</value>
   </data>
   <data name="AutodeskSamplePackage" xml:space="preserve">
-    <value>Autodesk sample</value>
+    <value>Sample View Extension</value>
```

Comment:
I think Roberto removed this resource string is his PR, so we may need to wait for his PR get merged first to make such change